backend/webdav/webdav.go

```diff
@@ -86,6 +86,10 @@ func init() {
 			Name:     "bearer_token_command",
 			Help:     "Command to run to get a bearer token",
 			Advanced: true,
+		}, {
+			Name:     "nextcloud_chunk_size",
```

Comment:
This should be called `chunk_size` I think, in the hope it will apply to other providers later :-)

Reply:
✅